builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -0,0 +1,1062 @@
+#!groovy
+
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurper
+import groovy.transform.Field
+
+// To be replaced as @Field def repo_credential_id = "value" for repo_credential_id, repo_base and repo_core
+
+@Field def repo_credential_id = "jazz_repocreds"
```

Comment:
Remove the values. These will be injected dynamically. 

Similar comment for other values initialized in this file.

Reply:
Removing four values for the variables that will be substituted dynamically as follows: repo_credential_id, repo_base, repo_core, scm_type

Reply:
Resolved with https://github.com/tmobile/jazz/pull/829/commits/ed1bfc77a34ecd5772e0511e89ad3b43faba5924